tests/basic/example-iptables-protection.ref

```diff
@@ -0,0 +1,36 @@
+{
+  "answerElements" : [
+    {
+      "class" : "org.batfish.datamodel.FlowHistory"
+    }
```

Comment:
Is this supposed to be empty? What about the other reachability answers?

Reply:
@jkhourybbn?

Reply:
Yes all three questions are supposed to be empty. A non empty result is a violation of the policy. The questions are crafted so they return no results.